Utilities/DataCompression/include/DataCompression/TruncatedPrecisionConverter.h

```diff
@@ -18,32 +18,29 @@
 //  @author Matthias Richter
 //  @since  2015-08-08
 //  @brief  A simple converter producing truncated precision
-//          according to a data model 
-
-namespace AliceO2 {
-
-template<class _ParameterModel>
-class TruncatedPrecisionConverter {
-public:
+//          according to a data model
+
+namespace AliceO2
+{
+template <class _ParameterModel>
+class TruncatedPrecisionConverter
+{
+ public:
```

Comment:
Doesn't that means that it's commented out? @matthiasrichter can you try rerunning with it enabled (try also `2`, rather than `-2` AFAICT that's what you need).